.github/workflows/comment-labeled.yml

```
@@ -10,6 +10,12 @@ env:
     This issue/PR was marked as stalled, it will be automatically closed in 30 days.
     If it should remain open, please leave a comment explaining why it should remain open.
   FAST_TRACK_MESSAGE: Fast-track has been requested by @${{ github.actor }}. Please 👍 to approve.
+  NOTABLE_CHANGE_MESSAGE: |
+    The notable-changes label has been added by @${{ github.actor }}.
+    Please include your text below if you'd like to include a more detailed summary in the changelog.
+    ```
+    [Insert here]
+    ```
```

```suggestion
    The https://${{ github.repository }}/labels/notable-change label has been added by @${{ github.actor }}.
    Please suggest a text for the release notes if you'd like to include a more detailed summary in the changelog, then proceed to update the PR description with the text or a link to the notable change suggested copy.
```

Something along these lines. Feel free to play around with it.

---

@panva when I opened the PR, the `event` provided a different url (https://github.com/danielleadams/node/pull/4#issuecomment-1475049364), so going to replace with the hard coded link.

---

Sure.